internal/pkg/cli/app_upgrade.go

```diff
@@ -81,11 +87,11 @@ func (o *appUpgradeOpts) Execute() error {
 		}
 		o.prog.Stop(log.Ssuccessf(fmtAppUpgradeComplete, color.HighlightUserInput(o.name), color.Emphasize(deploy.LatestAppTemplateVersion)))
 	}()
-	app, err := o.store.GetApplication(o.name)
+	err = o.upgradeApplication(app, version, deploy.LatestAppTemplateVersion)
```

Comment:
nit: the previous `return o.upgradeApplication(...)` looked fine to me 💭

Reply:
lol that won't work for this logic. https://github.com/aws/copilot-cli/pull/2178/files#diff-818ffee37657baf062cbd9badb1311ba36b1fbe89ac9b203edcf969d3ba4dc67R84 it will result in a success progress since we didn't set the `err` var